test/e2e/utils.go

```diff
@@ -841,25 +841,25 @@ func etcdZeroDownTimeValidatorJob(etcdSvc, testName string, tls *v1alpha1.TLSCon
 					VolumeSource: corev1.VolumeSource{
 						Secret: &corev1.SecretVolumeSource{
 							SecretName:  tls.TLSCASecretRef.Name,
-							DefaultMode: pointer.Int32(420),
+							DefaultMode: pointer.Int32(0640),
```

Comment:
Could this be reflected for all references in this file?
https://github.com/gardener/etcd-druid/pull/777#discussion_r1592016707